### PR TITLE
Updated screen shots with latest code; fixed typo; when creating a ne…

### PR DESCRIPTION
**Pull request by PeterSmithRedmond**
Updated the SetJobPropertyControl so that custom HTTP headers can be added. Like the other properties, these are always present if when they are not relevant (e.g., because the user is running on a very old system, or because the actual transfers are via SMB). The UI matches in principle how CURL works: multiple headers are added individually.

In addition, a typo was found and fixed in a comment in the quick file download dialog, and the screen shots were updated.

### CODE REVIEW

File: BITSManager/SetJobPropertyControl.xaml.cs [hunk 1]
```diff
@@ -52,6 +54,15 @@ public void SetJobProperties(BITS.IBackgroundCopyJob job)
                 }
             }
 
+            if (jobHttpOptions != null)
+            {
+                var text = _uiCustomHeadersAll.Text;
+                if (!String.IsNullOrWhiteSpace (text))
```
Comment (Diego-Perez-Botero) [inline BITSManager/SetJobPropertyControl.xaml.cs]: > ( [](start = 46, length = 2)

Nit: there shouldn't be any whitespace before the parenthesis. Same in line 62

File: BITSManager/SetJobPropertyControl.xaml [hunk 2]
```diff
@@ -49,5 +49,19 @@ Licensed under the MIT License.
...
         <TextBlock Text="{x:Static p:Resources.JobPassword}" x:Name="_uiJobPasswordHeader" />
         <TextBox x:Name="_uiPassword" HorizontalAlignment="Stretch" AutomationProperties.LabeledBy="{Binding ElementName=_uiJobPasswordHeader}" />
+        <Grid>
+            <Grid.ColumnDefinitions>
+                <ColumnDefinition Width="*" />
+                <ColumnDefinition Width="auto" />
+                <ColumnDefinition Width="auto" />
+            </Grid.ColumnDefinitions>
+            <TextBlock Text="{x:Static p:Resources.JobCustomHeadersTitle}" x:Name="_uiJobCustomHeadersHeader" />
+            <Button Grid.Column="1" Content="{x:Static p:Resources.JobAddHeader}" ToolTip="{x:Static p:Resources.JobAddHeaderToolTip}" Click="OnAddNewCustomHeader"  />
```
Comment (dabbers) [inline BITSManager/SetJobPropertyControl.xaml]: >ToolTip [](start = 82, length = 7)

Is there a reason for the tooltip? It doesn't exist on the other button and I Don't see tooltips used anywhere else in this file. Seems kinda random to just start adding tooltips all of a sudden.

**Comment by dabbers** [comment]
For the quick file download image, the more properties should be expanded (like the old picture).

**Comment by Diego-Perez-Botero** [comment]
Nit: the "Custom Http Headers" content in the "Job Details" panel should begin in the line below the "Custom HTTP Headers" label. For jobs with multiple headers, it looks weird that the first header is misaligned with respect to the rest.

File: BITSManager/SetJobPropertyControl.xaml (hunk 2 above)
Comment (PeterSmithRedmond) [reply BITSManager/SetJobPropertyControl.xaml]: The other button ought to have a tooltip, too. all buttons other than standard ones like "OK" and "Cancel" and "open" should have them (it helps with accessibility)

---
In reply to: [248038002](https://github.com/Microsoft/BITS-Manager/pull/11#discussion_r248038002) [](ancestors = 248038002)

File: BITSManager/SetJobPropertyControl.xaml.cs (hunk 1 above)
Comment (PeterSmithRedmond) [reply BITSManager/SetJobPropertyControl.xaml.cs]: I asked Visual Studio to reformat the entire file, and the parenthesis now follow the correct standard. This is the only human-edited .CS file in the PR, so there aren't any other files to fix up.

---
In reply to: [248035881](https://github.com/Microsoft/BITS-Manager/pull/11#discussion_r248035881) [](ancestors = 248035881)

**Comment by PeterSmithRedmond** [comment]
Fixed!

---
In reply to: [454559938](https://github.com/Microsoft/BITS-Manager/pull/11#issuecomment-454559938) [](ancestors = 454559938)

**Comment by PeterSmithRedmond** [comment]
The original image wasn't "expanded" -- it was the old version that didn't include the expando-properties. We don't currently need an expanded screen shot

---
In reply to: [454526752](https://github.com/Microsoft/BITS-Manager/pull/11#issuecomment-454526752) [](ancestors = 454526752)

**Pull request merged**